vmd-api/api/openapi.yaml

```diff
@@ -0,0 +1,336 @@
+openapi: 3.1.0
+info:
+  contact:
+    email: nikita.bazulin@unikie.com
+  description: This is Control Panel API based on the OpenAPI 3.1 specification of
+    the GHAF project
+  license:
+    name: Apache License 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  title: Virtual Machine Manager API of the GHAF project - OpenAPI 3.1
```

Comment:
s/Manager/Management/

Reply:
s/of the GHAF project/on the Ghaf platform/

Reply:
Done